html/includes/api_functions.inc.php

```diff
@@ -1693,3 +1695,34 @@ function list_logs()
     $app->response->headers->set('Content-Type', 'application/json');
     echo _json_encode($output);
 }
+
+function validate_column_list($columns, $tableName)
+{
+    global $config;
+    $app = \Slim\Slim::getInstance();
```

Comment:
Probably only need to get an instance of slim if there is an error, so you could move this into that if block.